recipes/cleopatra/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "cleopatra" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MAfarrag/cleopatra/archive/{{ version }}.tar.gz
+  sha256: 1c59d6e8d0826f25cfadc5dab2e8582a14fae224dfb12502f784172cc9b6ac55
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python  >=3.7.1,<3.10
+    - pip >=21.3.1
+  run:
+    - python  >=3.7.1,<3.10
+    - pip >=21.3.1
```

Comment:
Is pip actually a runtime dependency or does this come from the fact that the environment file on the repo uses it to install more?

Reply:
yes because i use it in the environment file in the repo